src/App/Command/SlackNotifierCommand.php

```diff
@@ -784,14 +770,14 @@ protected function checkStatsQA(): string
             $cache[$dateJSub1] = $cache[date('Y-m-d')];
         }
 
-        $diff = $countPR176 - $cache[$dateJSub1]['176'];
-        $countPR176Diff = isset($cache[$dateJSub1], $cache[$dateJSub1]['176'])
-            ? ' (' . ($diff == 0 ? '=' : ($diff > 0 ? '+' : '') . $diff) . ')'
-            : '';
         $diff = $countPR177 - $cache[$dateJSub1]['177'];
         $countPR177Diff = isset($cache[$dateJSub1], $cache[$dateJSub1]['177'])
             ? ' (' . ($diff == 0 ? '=' : ($diff > 0 ? '+' : '') . $diff) . ')'
             : '';
+        $diff = $countPR178 - $cache[$dateJSub1]['178'];
```

Comment:
Same, if you use const, you can make it configurable easily :) and never had to change it in the future

Reply:
Actually, I must remove the cache as we can't use it in Github Actions like Travis CI.

Reply:
Fixed